src/Gateways/Requests/AbstractRequest.php

```diff
@@ -61,7 +73,11 @@ public function get_default_args() {
 				'postalCode'     => $this->order->get_billing_postcode(),
 				'country'        => $this->order->get_billing_country(),
 			) : null,
-			RequestArg::CARD_HOLDER_NAME => null !== $this->order ? $this->order->get_formatted_billing_full_name() : null,
+			RequestArg::CARD_HOLDER_NAME =>
+				$this->get_card_holder_name(
+					null !== $this->order ? $this->order->get_formatted_billing_full_name() : null,
+					$this->data
```

Comment:
I still see the billing name in the requests, not the input from the hf.

Reply:
The issue here is that there are many request running on here. To debug it, i had to log every request and then  i could change it to the right one and get the data .